core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -210,6 +210,7 @@
         virtualHostTemplate.tlsAllowUnsafeCiphers(false);
         virtualHostTemplate.annotatedServiceExtensions(ImmutableList.of(), ImmutableList.of(),
                                                        ImmutableList.of());
+        virtualHostTemplate.blockingTaskExecutor(blockingTaskExecutor, shutdownBlockingTaskExecutorOnStop);
```

Comment:
We have to remove the `blockingTaskExecutor` and `shutdownBlockingTaskExecutorOnStop` field from this class and set the default values directly here.
```suggestion
        virtualHostTemplate.blockingTaskExecutor(CommonPools.blockingTaskExecutor(), false);
```